### DISCUSSION TITLE
sls remove for database/dynamodb-single-table won't delete table

### DISCUSSION THREAD

**Discussion by ianzone** [getlift/lift#155] [General]
is this a feature or bug? if it's a feature, how can I enable table deletion when running sls remove?

**Comment by t-richard** [comment]
It's on purpose and the docs say so (even though it could be clearer).

> You however won't loose any data as all tables are configured to be left as is when removed from a CloudFormation template.

It's probably what you want in production to avoid mistakes but it could be interesting to have an option to change this for dev and/or ephemeral environments.

**Reply by ianzone** [reply]
Thanks for explanation :)
